src/templates/digitalMedia/body/mediaAnnotations/MediaAnnotations.js

```diff
@@ -0,0 +1,80 @@
+import { Row, Col } from 'react-bootstrap';
+
+/* Import Sources */
+import DigitalMediaFilterLayer from 'sources/digitalMediaFilterLayer';
+
+
+const MediaAnnotaions = (props) => {
+    const annotations = props.annotations;
+
+    if (annotations) {
+        if (Object.keys(annotations).length > 0) {
+            return (
+                <Row className="h-100">
+                    <Col>
+                        <Row>
+                            <Col className="digitalMedia_annotationsBlockHeader px-4 py-2 fw-bold">
+                                <Row>
+                                    <Col md={{ span: 7 }}>
+                                        Attribute
+                                    </Col>
+                                    <Col md={{ span: 5 }}>
+                                        Type
+                                    </Col>
+                                </Row>
+                            </Col>
+                        </Row>
+
+                        <Row>
+                            <Col className="digitalMedia_annotationsBlock">
+                                {Object.keys(annotations).map((group, _i) => {
```

Comment:
I would place this map in a separate function

Reply:
Removed map function from jsx and moved to functional code, prepares array and then renders the components in the array in the view